src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -7,6 +7,7 @@
 #include "ActionArgs.h"
 #include "Command.h"
 
+#include <WinUser.h>
```

Comment:
Did we need this just for `VK_MENU`? IMO we might as well include it in the pch, since I doubt this is the last time we'll need it

Reply:
Yeah just for `VK_MENU`, I'll throw it in pch